acceptance/config_test.go

```diff
@@ -31,6 +31,10 @@ type TestConfig struct {
 	// If true, run this test when running with cloud env configured
 	Cloud *bool
 
+	// If true, this test is not run if -short is passed and cloud env is configured
+	// This implies Cloud = true.
+	CloudLong *bool
```

Comment:
Do we use `-short` in integration runs from a PR? That is, is this flag meant to annotate tests that run only in our nighlies?

Reply:
Yes.